src/main/kotlin/graphics/scenery/backends/vulkan/VulkanRenderer.kt

```diff
@@ -328,6 +329,7 @@ open class VulkanRenderer(hub: Hub,
     private var recordMovie: Boolean = false
     private var recordMovieOverwrite: Boolean = false
     override var pushMode: Boolean = false
+    val postRenderLambdas = ArrayList<()->Unit>()
```

Comment:
I know it was my naming originally, but postRenderLambdas is a bit cumbersome. Maybe we can rename them to something nicer?

Reply:
I actually quite like the name, from a user's point of view. Makes it quite clear what the functionality is. Do you have another suggestion?

Reply:
@skalarproduktraum please let me know if it is good as is so we can merge this